app/models/consignment_inquiry.rb

```diff
@@ -1,5 +1,6 @@
 class ConsignmentInquiry < ApplicationRecord
   validates :email, format: { with: URI::MailTo::EMAIL_REGEXP }
+  validates :recipient_email, format: { with: URI::MailTo::EMAIL_REGEXP }, allow_nil: true
```

Comment:
**question:** What happens if `recipient_email` is `nil` (because we're allowing `allow_nil: true` here)?

Reply:
On Pulse when recipient_email the inquiry will be delivered to sell@artsy.net

Reply:
Makes sense :)